src/cudafeatbin/compute-fbank-feats-cuda.cc

```diff
@@ -76,7 +76,7 @@ int main(int argc, char *argv[]) {
 
     std::string output_wspecifier = po.GetArg(2);
 
-    // Fbank is implemented via the MFCC code path 
+    // Fbank is implemented via the MFCC code path
```

Comment:
über-nit: period at end of sentence. [Yes, it's _that much_ strict](https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar)!